.travis.yml

```diff
@@ -39,6 +41,13 @@ script:
 - cargo run --release --bin rwlock -- 1 1 1 0 1 2
 - cd ..
 
+# REVIEW: needs a test that parking_lot still builds when structured like
+# libstd (aka including modules manually). This is easy to break by accident and
+# CI should help that import paths and such are always organized correctly.
+
+# REVIEW: needs at least one builder for all of `thread_parker` implementations
+# (at least building it, executing it would be best), missing ones are wasm/OSX
```

Comment:
I add macOS testing here: https://github.com/Amanieu/parking_lot/pull/147

But I'm not sure how we could test wasm. It requires the specially built libstd with atomic support, but we can't easily get that on travis, can we?

Reply:
There's not really a great way to test it via execution since engines haven't fully caught up yet all over the place, but I'd recommend using `xargo` plus `RUSTFLAGS` to ensure that the support here at least builds and produces a binary.